scripts/base/mallmadness.nut

```diff
@@ -0,0 +1,100 @@
+//
+
+weaponModelIDs <- [293, 259, 260, 261, 262, 263, 264, 265, 266, 267,
+                   268, 269, 270, 291, 271, 272, 273, 274, 275, 277,
+                   278, 279, 281, 282, 283, 284, 280, 276, 285, 286,
+                   287, 288, 289, 290];
+
+weaponPickupAmmo <- [ 1,  1,  1,  1,  1,  1,  1,  1,  1,  1,
+                      1,  1,  3,  3,  3,  3,  2, 10,  8,  8,
+                     12, 12, 30, 30, 30, 30, 30, 30, 14, 12,
+                      4, 50, 60, 60];
```

Comment:
This should be balanced.

Reply:
We shouldn't need this. If anything, it should probably be a factor for `GetAmmoAtSlot`. but for now 1.0 is fine.

We hopefully only have to balance the ammo that we initially spawn + maybe the overall amount of ammo that's on the server.